gmso/core/angle.py

```diff
@@ -35,6 +35,26 @@ def __init__(self, connection_members=[], connection_type=None, name="Angle"):
         super(Angle, self).__init__(connection_members=connection_members,
                 connection_type=connection_type, name=name)
 
+    def get_equivalent_members(self):
+        """Get a unique dataset representing the connection
+
+        Returns
+        _______
+        tuple
+            A unique tuple to represent the connection members
+
+        Notes
+        _____
+        For an angle:
+            i, j, k == k, j, i
+        where i, j, and k are the connection members.
+        """
+
+        return tuple([
+            self.connection_members[1],
+            frozenset([self.connection_members[0],
+                       self.connection_members[2]])
+            ])
```

Comment:
This is really good. Would it help to make this structure more accessible?  There could be other use cases for the equivalent connection members of a connection. This is tailored for use inside the `Topology` class dictionary key. However, if there are other utilities for this method, then it should probably return a tuple of all possible combinations of equivalencies, but should have an internal property that could then be levaraged by `Topology` class.

Reply:
So do you think `self.get_equivalent_members()` should return something like this:
```python3
((self.connection_members[0], self.connection_members[1], self.connection_members[2]),
 (self.connection_members[2], self.connection_members[1], self.connection_members[0]))
```
and then the current `get_equivalent_members()` should be renamed and become an internal method?